common/utils.c

```diff
@@ -449,6 +449,13 @@ void errbuffer_temp_high(uint8_t tm4c, uint8_t fpga, uint8_t ffly, uint8_t dcdc)
   return;
 }
 
+void errbuffer_volt_high(uint8_t tm4c, uint8_t dcdc)
+{
+  errbuffer_put(EBUF_CONTINUATION, tm4c);
```

Comment:
CONTINUATION is intended for codes that are long; I think you need to have at least one part that indicates what the code you are continuing is.

Reply:
do you mean I should have this[ line](https://github.com/apollo-lhc/cm_mcu/blob/dev_alarm_tasks/projects/cm_mcu/AlarmUtilities.c#L224)?

but I was not sure if we need this [line](https://github.com/apollo-lhc/cm_mcu/blob/dev_alarm_tasks/projects/cm_mcu/AlarmUtilities.c#L230) as well because there is no error code for a normal voltage like [EBUF_TEMP_NORMAL](https://github.com/apollo-lhc/cm_mcu/blob/9e857dbfc0f63bcaacc9c7ef438004951c6fa1fd/common/utils.h#L58) for temperature alarm.